platforms/android/build_sdk.py

```diff
@@ -59,6 +59,7 @@ def check_executable(cmd):
     try:
         log.debug("Executing: %s" % cmd)
         result = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        result = str(result)
```

Comment:
Could you please use this?
```
        if not isinstance(result, str):
            result = result.decode("utf-8")
```

(similar to changes in `platforms\winpack_dldt\build_package.py`)

Reply:
All done.